main.tf

```diff
@@ -0,0 +1,48 @@
+
+resource "aws_dynamodb_table" "main-table" {
+  name           = var.table_name
+  billing_mode   =  var.billing_mode
+  read_capacity  =  var.read_capacity_main-table
+  write_capacity =  var.write_capacity_main-table
+  hash_key       =  var.hash_key
+  range_key      =  var.range_key
+
+
+  attribute {
```

Comment:
attribute should be dynamic. so that we can add more attribute according to user